scriptworker/task.py

```diff
@@ -229,16 +230,15 @@ def guess_content_type(path):
     }
     args = [get_task_id(context.claim_task), get_run_id(context.claim_task),
             target_path, payload]
+    _compress_if_text_plain(path, payload['contentType'])
```

Comment:
I would love asc files to be text/plain, so we can read the chain of trust artifacts in the browser without downloading first.  If gzipping doesn't affect the artifact when we download from the queue (i.e., we don't have to gunzip before we can read it), then having asc files gzipped sounds like a win.  If we have to gunzip, then we'd have to change the scriptworker.cot.verify.download_cot to detect when an asc file is gzipped and gunzip (docker-worker asc files may not be gzipped, for example).